src/MainWindow.vala

```diff
@@ -186,11 +186,18 @@ namespace Terminal {
             restore_saved_state (restore_pos);
 
             clipboard = Gtk.Clipboard.get (Gdk.Atom.intern ("CLIPBOARD", false));
-            update_context_menu ();
             clipboard.owner_change.connect (update_context_menu);
 
             primary_selection = Gtk.Clipboard.get (Gdk.Atom.intern ("PRIMARY", false));
 
+            var open_in_browser_menuitem = new Gtk.MenuItem ();
+            open_in_browser_menuitem.set_action_name (ACTION_PREFIX + ACTION_OPEN_IN_BROWSER);
+            open_in_browser_menuitem.add (
+                new Granite.AccelLabel.from_action_name (
+                    _("Open in Browser"), open_in_browser_menuitem.action_name
```

Comment:
I see how this is technically just a URI, but I don't think it's clear that the location will open in Files when this menuitem is triggered the majority of the time. This label feels like optimizing for the less-likely case that we're opening a web URL

I think this might be better off waiting for a version of Granite where we can easily update this label based on the URI scheme

Reply:
OK, seems like a plan.